onadata/apps/api/viewsets/open_data_viewset.py

```diff
@@ -138,11 +180,19 @@ def data(self, request, **kwargs):
 
             if should_paginate:
                 instances = self.paginate_queryset(instances)
+            # Remove metadata fields from the instance
+            for row in DataInstanceSerializer(instances, many=True).data:
```

Comment:
Could we exclude these fields within the serializer ? It would remove the need for us to manually filter them out here

Reply:
Moved that